mobile_back_apple/cpp/backend_coreml/main.cc

```diff
@@ -53,7 +85,8 @@ mlperf_status_t mlperf_backend_flush_queries(mlperf_backend_ptr_t backend_ptr) {
 
 // Return the number of inputs of the model.
 int32_t mlperf_backend_get_input_count(mlperf_backend_ptr_t backend_ptr) {
-  return -1;
+  NSLog(@"input %d", [[[((CoreMLBackendData *) backend_ptr)->model modelDescription] inputDescriptionsByName] count]);
+  return [[[((CoreMLBackendData *) backend_ptr)->model modelDescription] inputDescriptionsByName] count];
```

Comment:
According to my understanding this value should not change with batch size . The same with output_count aswell.
I'm commenting in this PR only to test my knowledge with backend integration.

Reply:
Yes. The input and output (tensor) count is fixed per benchmark / dataset. E.g 1 input and 1 output for ImageNet or 1 input and 4 outputs for COCO.
The batch size is just one dimension of the tensor, with value of `1` for Single Stream or `N` for Offline scenario.